protocol/src/lazy.rs

```diff
@@ -1,10 +1,15 @@
 use arc_swap::ArcSwap;
 use ckb_always_success_script::ALWAYS_SUCCESS;
-use ckb_types::{core::ScriptHashType, packed, prelude::*};
+use ckb_types::{bytes::Bytes, core::cell::CellMeta, core::ScriptHashType, packed, prelude::*};
 
 use crate::ckb_blake2b_256;
+use crate::traits::BYTE_SHANNONS;
 use crate::types::{Hex, MerkleRoot};
 
+use std::sync::OnceLock;
+
+static CELL: OnceLock<CellMeta> = OnceLock::new();
```

Comment:
Why use `OnceLock` rather than `lazy_static!`.

Reply:
The trend of the times, lazy static has completed its historical role, there are some types in std, I recommend using std in the future